lib/index_column.c

```diff
@@ -744,6 +747,9 @@ static void
 grn_index_column_diff_process_token_id(grn_ctx *ctx,
                                        grn_index_column_diff_data *data)
 {
+  if (data->current_source.n_tokens >= GRN_II_MAX_TF) {
+    return;
+  }
```

Comment:
early returnで次の処理と関連はないので、空行を入れて次の処理と関係ないことを示しましょうか。

```suggestion
  }

```

Reply:
fix: [4f3c17a](https://github.com/groonga/groonga/pull/2035/commits/4f3c17a4375b68563df3357d914260c56247d25a)